docs/dev_install.rst

```diff
@@ -132,22 +147,43 @@ Setting up Taichi for development
     #   cmake .. -DCUDA_VERSION=10.0 -DTI_WITH_CUDA:BOOL=True
```

Comment:
Btw, I thought this is no longer needed thanks to `cuda_driver.h`?

Reply:
You are right!